data.sql

```diff
@@ -7,3 +7,57 @@ VALUES
 ('Pikachu', '07-01-2021', 1, false, 15.04),
 ('Devimon', '12-05-2017', 5, true, 11)
 ;
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+('Charmander', '08-02-2020', 0, false, 11),
```

Comment:
- Hey, Please, let's make the value for the weight a floating number and not an integer here. `-11.0`